### PR TITLE
add goreleaser,nerdctl,lima

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

### CODE REVIEW

File: goreleaser.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  description: Deliver Go binaries as fast and easily as possible
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - apk-tools
```
Comment (imjasonh) [inline goreleaser.yaml:14]: I don't think these need apk-tools...

File: lima.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package:
+  name: lima
+  version: 0.14.2
+  epoch: 0
+  description: Linux virtual machines, typically on macOS, for running containerd
```
Comment (imjasonh) [inline lima.yaml:5]: Is lima usable/useful from inside a container on (presumably) Linux? I think you know more about lima than I do, are you planning something by supporting this in Wolfi?

File: goreleaser.yaml (hunk 1 above)
Comment (developer-guy) [reply goreleaser.yaml:14]: agree, yes.

File: lima.yaml (hunk 2 above)
Comment (developer-guy) [reply lima.yaml:5]: I'm not sure about that, good question 🤔 @akihirosuda can you please help me ?

**Comment by imjasonh** [comment]
Closing this for now as we're waiting to hear whether limactl makes sense inside a container itself. Feel free to reopen.

**Pull request closed without merging**